h/static/scripts/directive/via-dialog.coffee

```diff
@@ -0,0 +1,27 @@
+###*
+# @ngdoc directive
+# @name viaLinkDialog
+# @restrict A
+# @description The dialog that generates a via link to the page h is currently
+# loaded on.
+###
+module.exports = ['$timeout', 'crossframe', ($timeout, crossframe) ->
+    viaUrl = 'https://via.hypothes.is/h/'
```

Comment:
I don't think it's a good idea to hard-wire this URL here.
This repo is supposed to be generic; anybody can deploy H and VIA anywhere.
The URL should be configurable in a config file.